templates/service.mustache

```diff
@@ -22,31 +22,49 @@ export interface {{FUNCTION_NAME}}Params {
 	/**
 	 * {{FUNCTION_PARAM_DESCRIPTION}}
 	 */
-	{{FUNCTION_PARAM}}: string;
+	{{FUNCTION_PARAM}}{{^FUNCTION_PARAM_OPTIONAL}}?{{/FUNCTION_PARAM_OPTIONAL}}: string;
```

Comment:
- [ ] Is the swagger documented well enough that we won't see errors throughout with this?

Reply:
Properties not marked as required will default to optional in the generated content.

Reply:
Fields not marked as required will be set to optional in the generated code. It won't produce an error but can be misleading when implementing a call inside the web app that does require a field but was marked as optional if it was not well-documented in swagger. 

Can't check the checkmark boxes btw lol